### PR TITLE
Improvements to the tagging system

### PR DESCRIPTION
**Pull request by darrylmasson**
In this PR:
- Tags beginning with an underscore (`_bad`, `_commissioning`, etc) cannot be submitted via the website.
- Some changes to the tag recommendations (gone are `earthquake` and `nofield`, new `prioritize`, and replace `bad` with `abandon`)
Also removed all the vestigial code for the 1T runs table, and reindented several functions.

### CODE REVIEW

**Review by jorana — COMMENTED** [review]
is the `abandon` tag also protected? If not, shouldn't we use `_abandon`?

**Comment by darrylmasson** [comment]
`abandon` is not a protected tag. We can certainly discuss restricting to "authorized" users (similar to the logic around removing tags), but the logic currently prevents _all_ website users from adding underscore tags

**Comment by tunnell** [comment]
_protected tags should only be set by scripts I’d think.  Abandon for shifters isn’t so bad since they can only ruin one or two runs by misusing since only bootstrax listens?

Thanks Darryl for changes

Sent from phone

**Comment by darrylmasson** [comment]
@tunnell @jorana What's the verdict? This is ready to merge

File: views/runsui.pug [hunk 1]
```diff
@@ -18,11 +18,10 @@ block content
         div.form-group.row
           label(for="taginput") Add tags to selected: &nbsp;
           select.form-control#taginput(style="width:20%;min-width:150px") Choose a tag
+            option(value="prioritize") prioritize
             option(value="messy") messy
-            option(value="bad") bad
+            option(value="abandon!!!11one1!!!1") abandon
```
Comment (jorana) [inline views/runsui.pug:23]: can we not do this?

**Review by jorana — COMMENTED** [review]
I don't know java so my review is not very useful.

I don't see (I might have overlooked) any protection against abandoning data, which I guess is your take-away from our discussion on slack? I'm fine with that but I've at least expressed my concerns.

Perhaps to prevent the `abandon` tag from being added in rage-quit, could we only allow it for data last ~3 days (otherwise it's not useful anyway). I can also of course add such a requirement to bootstrax.

File: views/runsui.pug (hunk 1 above)
Comment (darrylmasson) [reply views/runsui.pug]: I blame @WenzDaniel

Comment (WenzDaniel) [reply views/runsui.pug]: :D

Comment (jorana) [reply views/runsui.pug]: I would have to add this to a bootstrax query, my eyes hurt to see this, perhaps we should have a dedicated NV tag where we can have Daniel have his moment of fun

**Comment by darrylmasson** [comment]
Adding a statute of limitations for `abandon` tags to bootstrax would be simpler than adding it here

**Pull request merged**